docs/topics/migration-paths.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/getting-started-guide/master.adoc
 
+:_content-type: REFERENCE
```

Comment:
@sbeskin-redhat @stoobie  I am a little hesitant about calling this module "reference" because it's something we want every user to read and not gloss over as "technical background." But you know our users better than me. What do you think?

Reply:
@RichardHoch  It's a list of supported migration paths - why isn't it reference?